backend/dataall/modules/mlstudio/api/enums.py

```diff
@@ -0,0 +1,11 @@
+"""Contains the enums GraphQL mapping for SageMaker ML Studio"""
+from dataall.api.constants import GraphQLEnumMapper
+
+
+class SagemakerStudioRole(GraphQLEnumMapper):
+    """Describes the SageMaker ML Studio roles"""
+
+    Creator = '950'
+    Admin = '900'
+    Shared = '300'
+    NoPermission = '000'
```

Comment:
Shared is not used at the moment. Should we remove it?

Reply:
If it's not used and never been used we can delete it both from `SagemakerStudioRole` `enum` and `GraphQLEnum`. But if it was used in the past and we still have records in the datasets we should keep it to avoid issues with the serialization

Reply:
At least since it was open-sourced it was never used so I am removing it

Reply:
@nikpodsh I alse removed the whole Enum class from `backend/dataall/db/models/Enums.py`